js/ui/windowManager.js

```diff
@@ -135,6 +137,76 @@ const DisplayChangeDialog = new Lang.Class({
     },
 });
 
+const EOSShellWobbly = new Lang.Class({
+    Name: 'EOSShellWobbly',
+    Extends: EndlessShellFX.Wobbly,
+    Properties: {
+        'settings' : GObject.ParamSpec.object('settings',
+                                              'Effect GSettings',
+                                              'Wobbly Effect GSettings',
+                                              GObject.ParamFlags.CONSTRUCT_ONLY |
+                                              GObject.ParamFlags.WRITABLE,
+                                              Gio.Settings)
+    },
+
+    _init: function(params) {
+        this.parent(params);
+        this._settings = null;
+    },
+
+    set settings (val)
+    {
+         if (val['schema-id'] !== 'org.gnome.shell')
```

Comment:
I don't see how anything different could happen here; you can remove this code (or just use the global settings here directly)